yupana/processor/report_slice_processor.py

```diff
@@ -186,6 +199,80 @@ def generate_upload_candidates(self):
                       for key in host.keys() if key not in ['cause', 'status_code']}
         return candidates
 
+    def _transform_os_release(self, host: dict):
+        """Transform 'system_profile.os_release' label."""
+        if 'os_release' in host['system_profile']:
+            os_release = host['system_profile']['os_release']
+            if isinstance(os_release, str):
+                if os_release and os_release.strip():
+                    match_result = OS_RELEASE_PATTERN.match(os_release)
+                    parsed_info = match_result.groupdict()
+                    os_version = parsed_info['version'].strip()
+                    LOG.info(
+                        format_message(
+                            self.prefix,
+                            "os version after parsing os_release: '%s'"
+                            % os_version,
+                            account_number=self.account_number,
+                            report_platform_id=self.report_platform_id))
+                    if os_version:
+                        if os_release != os_version:
+                            host['system_profile']['os_release'] = os_version
+                            LOG.info(
+                                format_message(
+                                    self.prefix,
+                                    "os_release transformed '%s' -> '%s'"
+                                    % (os_release, os_version),
+                                    account_number=self.account_number,
+                                    report_platform_id=self.report_platform_id)
+                            )
+                    else:
+                        del host['system_profile']['os_release']
+                        LOG.info(
+                            format_message(
+                                self.prefix,
+                                'Removed empty os_release fact',
+                                account_number=self.account_number,
+                                report_platform_id=self.report_platform_id))
+                else:
+                    del host['system_profile']['os_release']
+                    LOG.info(
+                        format_message(
+                            self.prefix,
+                            'Removed empty os_release fact',
+                            account_number=self.account_number,
+                            report_platform_id=self.report_platform_id))
+                OS_RELEASE_TRANSFORMED.labels(
+                    account_number=self.account_number).inc()
+        return host
+
+    def _transform_os_kernel_version(self, host: dict):
+        """Transform 'system_profile.os_kernel_version' label."""
+        system_profile_info = host.get('system_profile', dict())
+        os_kernel_version = system_profile_info.get('os_kernel_version', '')
+
+        if isinstance(os_kernel_version, str) and os_kernel_version:
```

Comment:
Short circuiting here too?

```suggestion
        if not (isinstance(os_kernel_version, str) and os_kernel_version):
          return host
```